internal/manager/run.go

```diff
@@ -329,9 +329,9 @@ func Run(
 		} else {
```

Comment:
https://github.com/Kong/kubernetes-ingress-controller/blob/4af80319f5a02d6f1e449f840cd096a88099b414/internal/manager/run.go#L328
Here is not changed.